src/inet/UDPEndPointImplLwIP.cpp

```diff
@@ -113,10 +120,7 @@ CHIP_ERROR UDPEndPointImplLwIP::BindImpl(IPAddressType addressType, const IPAddr
 
 CHIP_ERROR UDPEndPointImplLwIP::BindInterfaceImpl(IPAddressType addrType, InterfaceId intfId)
 {
-    // A lock is required because the LwIP thread may be referring to intf_filter,
-    // while this code running in the Inet application is potentially modifying it.
```

Comment:
Are the comments here wrong?

Reply:
yes, they don't apply anymore.

Reply:
Is that because `LwIPBindInterface` now does `udp_bind_netif` in a synchronized way?

Are we guaranteed that mUDP cannot change between `GetPCB` and the use of the pointer in `LwIPBindInterface`?